errno/errname.go

```diff
@@ -1018,6 +1018,8 @@ var MySQLErrName = map[uint16]*mysql.ErrMessage{
 	ErrAssertionFailed:               mysql.Message("assertion failed: key: %s, assertion: %s, start_ts: %v, existing start ts: %v, existing commit ts: %v", []int{0}),
 	ErrInstanceScope:                 mysql.Message("modifying %s will require SET GLOBAL in a future version of TiDB", nil),
 	ErrNonTransactionalJobFailure:    mysql.Message("non-transactional job failed, job id: %d, total jobs: %d. job range: [%s, %s], job sql: %s, err: %v", []int{2, 3, 4}),
+	ErrSettingNoopVariable:           mysql.Message("setting %s has no effect in TiDB", nil),
+	ErrGettingNoopVariable:           mysql.Message("variable %s has no effect in TiDB", nil),
```

Comment:
Maybe `setting variable ...` and `getting variable ...`? I was dazed for a second when I saw the errors. I was thinking which error came from `set ..` and which for `select @@...`.

Reply:
Setting a variable is only possible via `SET` syntax. Getting is not a MySQL verb - it could be in a SELECT statement (most likely) that someone is fetching the value, but it could also be in INSERT/UPDATE/DELETE context, so this is why it used generic wording.

Reply:
That makes sense.